### ISSUE TITLE
ReSources: Integrate the new module to import data

### ISSUE THREAD

**Issue by arunge** [Pandora-IsoMemo/resources#70]

**Comment by arunge** [comment]
The [beta version](https://isomemoapp.com/app/resources-beta) of [Resources 23.02.2](https://github.com/Pandora-IsoMemo/resources/pull/72) now uses the import module with Pandora/url access and merge/query feature.

@isomemo or @CarloCocozza Could you please check carefully if import of data is working as expected. It was quite challenging to integrate the checks (that were used before) into the new module. I had to add extra functionality to the import module because of batch imports and because Resources expects output being a matrix. Could you try typical and not typical data and tell me if the app crashes or if the error messages are fine?

**Comment by isomemo** [comment]
@arunge 
@CarloCocozza did you test this?

**Comment by CarloCocozza** [comment]
@arunge 
I am sorry, I might have missed this email.

@isomemo  I am not aware what this is about. Maybe we can check this together later.

**Comment by isomemo** [comment]
@arunge I am checking ReSources beta and the input of files is the old one. Not the Pandora module.

**Comment by arunge** [comment]
@isomemo Are you talking about the [online version](https://isomemoapp.com/app/resources-beta)? I do have access to the very new impurt module (3) :thinking: 

![image](https://user-images.githubusercontent.com/16759098/221885444-35e82d3b-2139-48ea-9801-98ab42e13389.png)

Could you try to clean your cache of the browser please?

**Comment by isomemo** [comment]
@arunge 

This is really strange. This is what I get:

![image](https://user-images.githubusercontent.com/46485054/222184088-e7eca06f-1225-4a45-864d-bc94c6d5a706.png)

**Comment by arunge** [comment]
@isomemo Did you maybe use the "Browse" input (1)? This is only for upload of .zip files containing a saved model. There I did not add the import module. Since the Import-Module does not understand .zip files. We would need to add this functionality of uploading a model and transferring all inputs to the app. Until now, I even did not think about this option. Is this required?

If yes, we must talk about the implementation.

I only added the import module to the buttons where it is possible to upload tables from .csv or xlsx, e.g. (2), (3) 

![image](https://user-images.githubusercontent.com/16759098/222378066-996a37d8-3385-4bd7-998b-965a40cd3aef.png)

![image](https://user-images.githubusercontent.com/16759098/222378764-e5dfd08c-add0-4189-a595-23f9ccbc8a1e.png)

**Comment by isomemo** [comment]
@arunge Yes, it should work for the Browse input here and for other apps! It should work for any type of uploaded files. However, only for certain types of files (e.g., txt, csv, Excel) will there be a data preview and data merger.

Remember also that this should work for saved SQL queries.

**Comment by arunge** [comment]
@isomemo Regarding

> Yes, it should work for the Browse input here and for other apps! It should work for any type of uploaded files.  However, only for certain types of files (e.g., txt, csv, Excel) will there be a data preview and data merger.
>
> Remember also that this should work for saved SQL queries.

this will be developed within
- https://github.com/Pandora-IsoMemo/iso-app/issues/145

So, I am closing here.
